### PR TITLE
Add VLC Remote Setup 3.70

### PR DESCRIPTION
**Pull request by alexreg**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/vlcremote-setup.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  desc "Set up VLC for VLC Remote"
+  homepage "https://hobbyistsoftware.com/VLC"
+
+  livecheck do
+    url "https://hobbyistsoftware.com/vlc-versions"
+    strategy :page_match
+    regex(%r{href=.*?/VLCSetup_(\d+(?:\.\d+)*)\.dmg}i)
+  end
+
+  app "VLC Setup.app", target: "VLC Remote Setup.app"
```
Comment (miccal) [inline Casks/vlcremote-setup.rb:16]: Why is this rename necessary? We generally would not do this unless there is a valid reason.

Comment (alexreg) [reply Casks/vlcremote-setup.rb:16]: I thought the name "VLC Setup" was rather confusing, given the name of the app everywhere else is "VLC Remote Setup" (optionally followed by "Helper"). I just saw this file in my `/Applications` dir today and thought it was something to do with VLC itself.

Comment (miccal) [reply Casks/vlcremote-setup.rb:16]: I agree that the naming of this is weird, but we need to follow upstream's naming here.

Comment (alexreg) [reply Casks/vlcremote-setup.rb]: What about putting it in a subdir of `/Applications`? If not, no worries, I'll just remove the `target` bit.

**Comment by miccal** [comment]
@alexreg could you please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap), thanks.

**Comment by alexreg** [comment]
@miccal Ah yes, sorry. Done now. I think everything is in order.

P.S. Not relevant to this PR, but was grepping the cask definitions, and it looks like `editaro` incorrectly specifies `ApplicationSupport` (without the space) when zapping. It's also missing a description. Just FYI. :-)

**Comment by alexreg** [comment]
That's very quick... thanks!
